index.html

```diff
@@ -8392,7 +8438,7 @@ <h3>Processing actions</h3>
    </dd>
   </dl>
 
- <li><p>Return (<var>x</var>, <var>y</var>)
+ <li><p>Return (<var>x</var> + <var>parentOffsetLeft</var>, <var>y</var> + <var>parentOffsetTop</var>)
```

Comment:
should the offset be applied to all origins or only the viewport? I think the origin pointer might already be in the top level context coordinates and not sure if the element origin does some adjustments already.

Reply:
That does not seem to follow from the spec text since input sources are per top-level browsing context.